services/route_manager/src/main/java/com/futurewei/alcor/route/dao/RouteTableRepository.java

```diff
@@ -83,6 +86,21 @@ public void addItem(RouteTable routeTable) throws CacheException {
         }
     }
 
+    @Override
+    public void addItems(List<RouteTable> items) throws CacheException {
+        try (Transaction tx = cache.getTransaction().start()) {
+            logger.log(Level.INFO, "Add route table batch: {}",items);
+            Map<String, RouteTable> routeTableMap = items.stream().collect(Collectors.toMap(RouteTable::getId, Function.identity()));
+            cache.putAll(routeTableMap);
+            tx.commit();
+        } catch (CacheException e) {
+            throw e;
+        } catch (Exception e){
+            logger.log(Level.FINE,"Add route table batch error",e);
```

Comment:
Same issue on exception handling. Refer to services/route_manager/src/main/java/com/futurewei/alcor/route/dao/RouteEntryRepository.java.